Project 3: Contact form (pair programming)

binhussen opened the pull request:
# Project 2: Contact form
In this section I have built for this milestone is:
- Added contact form in the footer.
- Created a form with a placeholder and validation based on requirement.
- I can able to collect data by using the [Formspree service](https://formspree.io/html).

zieeco commented on index.html:
```diff
@@ -353,6 +353,166 @@ <h2 class="skill-title">Skills</h2>
+              required
+            />
+          </fieldset>
+          <textarea name="message" maxlength="500" required>
+Enter text here
+            </textarea
```
**[REQUIRED CHANGE]**

- Please, kindly remove the text you added "Enter text here", as this is making your form submit when the Textarea input is empty, instead update by giving it a placeholder = "Enter text here".
```
<textarea name="message" id="message" rows="5" maxlength="500" placeholder="Enter text here" required></textarea>
```

zieeco replied:
**[REQUIRED CHANGES]**

*README FILE*

- Please, kindly update the getting started header and add some steps on how users can have your repo running locally on their machine.
- Also, update the authors by adding others that have contributed to the project.
- Kindly, remove the headers that are not relevant to the project to make your readme appear more professional.
- The live demo link should take users to the live demo view of your portfolio, it takes the user to your GitHub account which is not the expected outcome. Please, kindly update the live demo link or remove it, for now, to improve the user experience when checking your readme file and to make your readme appear more professional.

binhussen replied:
Thank you @zieeco for your feedback, I Will modify it.

zieeco reviewed (changes requested):
Hi @binhussen,


Great job so far! 👏 
You've done well implementing the contact form, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._


<img src="https://media0.giphy.com/media/8UGVQESpDMMQl7yd3F/giphy.gif"/>


## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

binhussen commented:
# Project 2: Contact form
In this section I have built for this milestone is:
- Updated Textarea tag by giving it a placeholder = "Enter text here".
- Updated the authors by adding others that have contributed to the project.
- Removed the headers that are not relevant to the project to make my readme appear more professional.
- Added a live demo link witch navigates to my portfolio.

danmainah commented on README.md:
```diff
@@ -8,7 +8,7 @@ a portfolio resume that showcases examples of my works along with the usual resu
 ## Live Demo
 
-[Live Demo Link](https://github.com/binhussen)
+[Live Demo Link](https://binhussen.github.io/portfolio/)
```
- [ ] [optional] @Team great work implementing the changes, it would be nice to ensure the live demo is the latest version of your app currently it doesn't have a contact section.

binhussen replied:
It is because the contact section waiting for approval to be merged into the main branch.

binhussen replied:
Thank you for your feedback, I will modify it.

danmainah reviewed (approved):
Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

binhussen commented:
Thank you @danmainah for your feedback and approval.
